Content.Server/GameObjects/Components/Weapon/Ranged/Barrels/ServerRangedBarrelComponent.cs

```diff
@@ -234,6 +234,7 @@ private void Fire(IEntity shooter, Vector2 targetPos)
             }
 
             // At this point firing is confirmed
+            ammo.SendMessage(this, new BarrelFiredMessage(projectile));
```

Comment:
Because I shitcoded FireProjectiles (gun pr fixes it I swear) you'll probably want this under FireProjectiles instead given for example shotgun shells might shoot out multiple times.